compiler/src/iree/compiler/Codegen/LLVMCPU/LLVMCPUVectorization.cpp

```diff
@@ -107,10 +110,78 @@ static SmallVector<int64_t> getCanonicalVectorShape(func::FuncOp funcOp) {
   return canonicalVectorShape;
 }
 
+/// Tries to infer the vector sizes from an IR that has been tiled for
+/// vectorization. Returns failure if vector sizes can't be inferred. For know
+/// this utility is only able to extract tile size information from
+/// `tensor.extract_slice` and `affine.min` ops, which should cover most but not
+/// all the cases.
```

Comment:
IIRC, it infers the shape using ValueBoundsConstraintSet, not the information from `tensor.extract_slice` and `affine_min` ops. Can you update the comment?